app/registration/views/patient-name-input.html

```diff
@@ -7,39 +7,89 @@ <h3 class="panel-title">{{'PATIENT_INFO_NAME' | translate}}</h3>
              ng-class="{'has-error': aForm.givenName.$invalid && showMessages}">
             <label for="givenName" class="col-sm-2 control-label">{{'PATIENT_INFO_NAME' | translate}}:<span class="asterick">*</span></label>
             <div class="col-sm-10">
-                <div ng-messages for="aForm.givenName.$error" 
+                <div ng-messages for="aForm.givenName.$error"
                      ng-messages-include="../poc-common/form-display/views/error-messages.html"
                      ng-show="showMessages">
                 </div>
-                <input type="text" class="form-control keyboard-in" id="givenName" name="givenName" 
+                <input type="text" class="form-control keyboard-in" id="givenName" name="givenName"
                        ng-model="patient.givenName"
                        placeholder="{{'PATIENT_INFO_NAME_PLACEHOLDER' | translate}}" title="Enter patient's first name."
                        my-autocomplete source="getAutoCompleteList" response-map="getDataResults"
                        data-item-type="personName"
                        required>
             </div>
         </div>
+        <div class="form-group form-group-lg"
```

Comment:
@ningosi is there any reason for changing this order? Normally we see *middle name* between *first name* and *surname*.